### PR TITLE
Update release notes draft to version v1.30.0-beta.0

### PR DESCRIPTION
**Pull request by rashansmith**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR updates the Release Notes Draft to k/k v1.30.0-beta.0

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
This is an automated PR generated from `krel The Kubernetes Release Toolbox`

### CODE REVIEW

File: releases/release-1.30/release-notes/maps/pr-123578-map.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+pr: 123578
+releasenote:
+  text: 'kubeadm: printed all the kubelets and nodes that need to be upgraded on "upgrade
```
Comment (npolshakova) [inline releases/release-1.30/release-notes/maps/pr-123578-map.yaml:3]:
```suggestion
  text: 'kubeadm: printed all the kubelets and nodes that need to be upgraded on "upgrade plan".'
```

File: releases/release-1.30/release-notes/release-notes-draft.json [hunk 2]
```diff
@@ -259,6 +259,24 @@
     ],
     "duplicate": true
   },
+  "120111": {
+    "commit": "20b28312ea9278c7b93340d0eca71023c45a43b3",
+    "text": "Adds the namespace when using 'kubectl logs \u003cpod-name\u003e' and the pod is not found. Previously the message returned would be 'Error from server (NotFound): pods \"my-pod-name\" not found'. This has been updated to reflect the namespace in the message as follows: 'Error from server (NotFound): pods \"my-pod-name\" not found in namespace \"default\"'",
```
Comment (npolshakova) [inline releases/release-1.30/release-notes/release-notes-draft.json:264]:
```suggestion
    "text": "Added the namespace when using 'kubectl logs \u003cpod-name\u003e' and the pod is not found. Previously the message returned would be 'Error from server (NotFound): pods \"my-pod-name\" not found'. This has been updated to reflect the namespace in the message as follows: 'Error from server (NotFound): pods \"my-pod-name\" not found in namespace \"default\"'",
```

File: releases/release-1.30/release-notes/release-notes-draft.json [hunk 3]
```diff
@@ -3916,5 +4604,857 @@
...
+      "feature"
+    ],
+    "sigs": [
+      "cluster-lifecycle"
+    ],
+    "feature": true
+  },
+  "123593": {
+    "commit": "89cbd94e68ae80414dea7c14c1694e5e378a188d",
+    "text": "It is possible to configure the IDs that the Kubelet uses to create user namespaces.\n\n\nUser namespaces support is a Beta feature now.",
```
Comment (npolshakova) [inline releases/release-1.30/release-notes/release-notes-draft.json:5015]: Should this have the multiple new lines here?

File: releases/release-1.30/release-notes/maps/pr-123641-map.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+pr: 123641
+releasenote:
+  text: 'kube-apiserver: the StructuredAuthorizationConfiguration feature gate is
```
Comment (ramrodo) [inline releases/release-1.30/release-notes/maps/pr-123641-map.yaml:3]:
```suggestion
  text: 'kube-apiserver: the StructuredAuthorizationConfiguration feature gate was
```

File: releases/release-1.30/release-notes/maps/pr-123641-map.yaml [hunk 5]
```diff
@@ -0,0 +1,4 @@
...
+    promoted to beta and allows using the `--authorization-configuration` flag.'
```
Comment (ramrodo) [inline releases/release-1.30/release-notes/maps/pr-123641-map.yaml:4]:
```suggestion
    promoted to beta and now allows using the `--authorization-configuration` flag.'
```

**Review by ramrodo — COMMENTED** [review]
Great!
Just a couple of suggestions

File: releases/release-1.30/release-notes/maps/pr-123578-map.yaml (hunk 1 above)
Comment (rashansmith) [reply releases/release-1.30/release-notes/maps/pr-123578-map.yaml:3]: Good catch!

File: releases/release-1.30/release-notes/release-notes-draft.json (hunk 3 above)
Comment (rashansmith) [reply releases/release-1.30/release-notes/release-notes-draft.json:5015]: @npolshakova Good catch. I think one new line if fine here

**Comment by rashansmith** [comment]
> Great! Just a couple of suggestions



> Great! Just a couple of suggestions

Thanks, I've applied the changes!

**Review by npolshakova — APPROVED** [review]
LGTM 🎉

**Pull request merged**